packages/lesswrong/components/users/account/UsersEditForm.tsx

```diff
@@ -915,6 +916,19 @@ const UsersForm = ({
         </div>
         </HighlightableField>}
 
+        {isEAForum &&
```

Comment:
The other fields have a `HighlightableField` wrapper, which is what makes it actually highlight it rather than just opening the section